### PR TITLE
feat(modules/overlay): overlay-transparent-API

### PR DESCRIPTION
**Pull request by yteslenk**
fix(modules/overlay): make overlay transparent. Hide controls and description.

### CODE REVIEW

File: src/modules/ui/overlay/overlay.ts [hunk 1]
```diff
@@ -94,7 +110,14 @@ class Overlay implements IOverlay {
     }
   }
 
+  private _showControlsAndDescription(){
```
Comment (BrooklynKing) [inline src/modules/ui/overlay/overlay.ts]: better `showMainUIBlock`. but if we gonna remove `this._isControlsAndDescriptionHidden`, then there are no reasons for this method at all

File: src/modules/ui/overlay/overlay.ts [hunk 2]
```diff
@@ -55,6 +61,10 @@ class Overlay implements IOverlay {
       this.hide();
     }
 
+    if (!config.hideOverlay) {
+      this._mainUIBlock.hide();
```
Comment (BrooklynKing) [inline src/modules/ui/overlay/overlay.ts]: use `disableShowingContent\enableShowingContent` from `mainUIBlock` public API

File: src/modules/ui/overlay/overlay.ts [hunk 3]
```diff
@@ -36,16 +39,19 @@ class Overlay implements IOverlay {
...
     engine: IPlaybackEngine;
     rootContainer: IRootContainer;
     theme: IThemeService;
     config: IPlayerConfig;
+    mainUIBlock: IMainUIBlock;
   }) {
     this._eventEmitter = eventEmitter;
     this._engine = engine;
     this._theme = theme;
+    this._mainUIBlock = mainUIBlock;
```
Comment (BrooklynKing) [inline src/modules/ui/overlay/overlay.ts]: Also we need to hide `loader` module

File: src/modules/ui/overlay/overlay.ts [hunk 4]
```diff
@@ -55,6 +67,10 @@ class Overlay implements IOverlay {
       this.hide();
     }
 
+    if (!config.hideOverlay) {
+      this._mainUIBlock.disableShowingContent();
```
Comment (BrooklynKing) [inline src/modules/ui/overlay/overlay.ts]: why you not hiding loader here?

Comment (yteslenk) [reply src/modules/ui/overlay/overlay.ts]: After class Overlay initialized, Overlay's private method _updatePlayingState called and invoked
_tryShowContent() method, in which loader hiding.

**Comment by BrooklynKing** [comment]
Last one - pls change commit prefix from `fix` to `feat`

**Pull request merged**